Library/Homebrew/tap_migrations.rb

```diff
@@ -22,14 +22,17 @@
   "cmucl" => "homebrew/binary",
   "comparepdf" => "homebrew/boneyard",
   "connect" => "homebrew/boneyard",
+  "coremod" => "homebrew/boneyard",
```

Comment:
I've gone ahead and put this back in boneyard, whilst adding the additional libxmp-lite formula.